src/main/java/co/cask/plugin/format/MultiTableDBInputFormat.java

```diff
@@ -74,21 +76,22 @@
       DatabaseMetaData dbMeta = connection.getMetaData();
       ResultSet tables = dbMeta.getTables(null, dbConf.getSchemaNamePattern(), dbConf.getTableNamePattern(),
           new String[] {"TABLE", "TABLE_SCHEM"});
-      Collection<TableInfo> tableInfos = new ArrayList<TableInfo>();
+      Collection<TableInfo> tableInfos = new ArrayList<>();
       List<DBTableSplit> splits = new ArrayList<>();
       List<String> whiteList = dbConf.getWhiteList();
       List<String> blackList = dbConf.getBlackList();
       while (tables.next()) {
         String tableName = tables.getString("TABLE_NAME");
+        // this is required for oracle apparently? Don't know why
         String db = tables.getString("TABLE_SCHEM");
```

Comment:
Not a part of this PR but is this SCHEM? Or SCHEMA?

Reply:
it's correct as SCHEM. The jdbc API is kind of funny.